eventually/curriculum/models.py

```diff
@@ -80,8 +80,6 @@ def create(name, owner, description='', goals=()):
         :type owner: CustomUser model
 
         :return: Curriculum object or nonenew_c = Curriculum.objects.creat(name='curr',
-         description='dededede', goals='sadsadsa', owner=11)
```

Comment:
hmm, what should be at the end of this line -> "Curriculum.objects.creat(name='curr', " ?